changelogs/fragments/5812-implement-updateconf-api-call.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - one_vm - implement the ``one.vm.updateconf`` API call (https://github.com/ansible-collections/community.general/pull/5812).
```

Mentioning the option makes this more useful for users:
```suggestion
  - one_vm - add a new ``updateconf`` option which implements the ``one.vm.updateconf`` API call (https://github.com/ansible-collections/community.general/pull/5812).
```